snippets/consumer_widgets.code-snippets

```diff
@@ -4,8 +4,8 @@
     "prefix": "stlessConsumer",
     "description": "Create a ConsumerStatelessWidget",
     "body": [
-      "class $1 extends ConsumerWidget {",
-      "\tconst $1({Key? key}) : super(key: key);",
+      "class ${1:name} extends ConsumerWidget {",
```

Comment:
I am pretty new in general to creating snippets, what is the difference between defining it like `1:name` and `1`?

Reply:
Name works as a placeholder here. When you write on top oof it, it disappears. I find it a bit more instructive, but let me know if your prefer it without the placeholder. 

https://user-images.githubusercontent.com/57945332/151124941-c717a893-f9a3-43b0-aa4b-aafa9d681623.mov

Reply:
Oh that is neat and something we should consider for all the other snippets as well.

Kinda hesitant to do it in this change though and think it would be better in a standalone PR where all the snippets get updated with the same convention.

So I propose that we do it the "normal" way in this change and if you want, you can make a new PR just updating all  snippets with this new convention!